corehq/apps/userreports/expressions/__init__.py

```diff
@@ -2,6 +2,15 @@
 from django.utils.module_loading import import_string
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 
+
+def get_custom_ucr_expressions():
+    custom_ucr_expressions = settings.CUSTOM_UCR_EXPRESSIONS
+
+    for expression_list in settings.CUSTOM_UCR_EXPRESSION_LISTS:
+        custom_ucr_expressions += import_string(expression_list)
+
+    return custom_ucr_expressions
```

Comment:
copying over comment from https://github.com/dimagi/commcare-hq/pull/10467:

is this not calling `import_string` on the whole list? also, even if that works won't the first half of the list be string tuples and the second half be imported modules? finally, if `CUSTOM_UCR_EXPRESSIONS` is a list and we add to it won't that get added over and over again each time?

```python
>>> x = [1, 2, 3]
>>> y = x
>>> y += [4, 5]
>>> x
[1, 2, 3, 4, 5]
>>> y += [4, 5]
>>> x
[1, 2, 3, 4, 5, 4, 5]
>>> 
```